apps/fluent-tester/src/FluentTester/TestComponents/MenuButton/MenuButtonTest.tsx

```diff
@@ -80,12 +82,156 @@ const menuButton: React.FunctionComponent<{}> = () => {
   );
 };
 
+const nestedMenuButton: React.FunctionComponent<{}> = () => {
+  const testImage = require('../Button/icon_24x24.png');
+
+  const svgProps: SvgIconProps = {
+    src: TestSvg,
+    viewBox: '0 0 500 500',
+  };
+
+  const [lastMenuItemClicked, setLastMenuItemClicked] = React.useState(null);
+
+  const [focusOnMount, setShouldFocusOnMount] = React.useState(true);
+  const toggleFocusOnMount = React.useCallback((value) => setShouldFocusOnMount(value), [setShouldFocusOnMount]);
+
+  const [focusOnContainer, setShouldFocusOnContainer] = React.useState(false);
+  const toggleFocusOnContainer = React.useCallback((value) => setShouldFocusOnContainer(value), [setShouldFocusOnContainer]);
+
+  const onItemClick = React.useCallback(
+    (key) => {
+      setLastMenuItemClicked(key);
+    },
+    [setLastMenuItemClicked],
+  );
+
+  const [showSubmenu, setShowSubmenu] = React.useState(false);
+  const [isSubmenuVisible, setIsSubmenuVisible] = React.useState(false);
+  const [lastSubmenuMenuItemClicked, setSubmenuLastMenuItemClicked] = React.useState(null);
+
+  const toggleShowSubmenu = React.useCallback(() => {
+    setShowSubmenu(!showSubmenu);
+    setIsSubmenuVisible(!isSubmenuVisible);
+  }, [showSubmenu, isSubmenuVisible, setShowSubmenu, setIsSubmenuVisible]);
+
+  const onShowSubmenu = React.useCallback(() => {
+    setIsSubmenuVisible(true);
+  }, [setIsSubmenuVisible]);
+
+  const onDismissSubmenu = React.useCallback(() => {
+    setShowSubmenu(false);
+  }, [setShowSubmenu]);
+
+  const onSubmenuItemClick = React.useCallback(
+    (key) => {
+      setSubmenuLastMenuItemClicked(key);
+    },
+    [setSubmenuLastMenuItemClicked],
+  );
+
+  const menuItems = [
+    {
+      itemKey: '1',
+      text: 'Menu item with png Icon',
+      icon: { testImage },
+    },
+    {
+      itemKey: '2',
+      text: 'Menu item',
+    },
+    {
+      itemKey: '3',
+      text: 'Disabled Menu Item',
+      disabled: true,
+    },
+    {
+      hasSubmenu: true,
+      itemKey: '4',
+      text: 'SubmenuItem svg icon',
+      componentRef: React.useRef(null),
+      icon: { svgSource: svgProps, width: 20, height: 20, color: 'red' },
```

Comment:
I don't think this "red" prop gets applied to the svg - svg icons get their color through the svgSource as they don't support PlatformColor yet. If the red isn't something that matters you can leave this prop out

Reply:
As it's a test if doesn't matter. I'll remove it